scripts/examples/gn_efr32_example.sh

```diff
@@ -226,6 +227,8 @@ else
     BUILD_DIR=$OUTDIR/$SILABS_BOARD
     echo BUILD_DIR="$BUILD_DIR"
     if [ "$USE_WIFI" == true ]; then
+        optArgs+="chip_enable_wifi_ipv4=$SL_WIFI_ENABLE_IPV4 "
+        optArgs+="chip_inet_config_enable_ipv4=$SL_WIFI_ENABLE_IPV4 "
```

Comment:
Why it is done this way now? Can't the default value just be taken care by the default value set in gn args files?

I would suggest instead doing just 
`optArgs+="chip_enable_wifi_ipv4=true  chip_inet_config_enable_ipv4= true"`

under the --chip_enable_wifi_ipv4) option. Removing the need for `SL_WIFI_ENABLE_IPV4=true` and not remove it from the` if [ "$USE_WIFI" == true ];`

Reply:
Would you believe me if I said that I had tried and it was not working, now it is working.